src/types.ts

```diff
@@ -19,22 +19,22 @@ export type User = {
   };
 };
 
-export interface IUserConfig { }
+export interface IUserConfig {}
 
 export interface IDetector {
   id: string;
   detector_id: string;
   detector_name: string;
   detector_config: IDetectorConfig;
+  char_num: string;
```

Comment:
Why ? This change broke the whole system design, if you want to do something like this, pls talk with me about it. Is it necessary ?

Reply:
You got a point there. I changed the logic. I kept the Char_Num and the Coma_Position inside the IDetectorConfig.